aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java

```diff
@@ -344,23 +347,17 @@ public void testDkim() throws Exception {
             assertTrue(attributes.getDkimVerificationStatus().equals("Pending"));
             assertTrue(attributes.getDkimTokens().size() == dkim.getDkimTokens().size());
 
-            for (String token1 : attributes.getDkimTokens()) {
-                boolean found = false;
-                for (String token2 : dkim.getDkimTokens()) {
-                    if (token1.equals(token2)) {
-                        found = true;
-                        break;
-                    }
-                }
-                assertTrue(found);
+            List verifyDomainResultTokens = dkim.getDkimTokens();
+            for (String token : attributes.getDkimTokens()) {
+                assertTrue(verifyDomainResultTokens.contains(token));
```

Comment:
This is so much more succinct. 
Minor suggestion: assertTrue(dkim.getDkimTokens().containsAll(attributes.getDkimTokens()))?

Reply:
TIL Java List has a `containsAll` method.

Done.